src/lib/mongoBackend/MongoGlobal.cpp

```diff
@@ -251,9 +251,18 @@ bool getOrionDatabases(std::vector<std::string>* dbsP)
 
     if (strncmp(prefix.c_str(), dbName.c_str(), strlen(prefix.c_str())) == 0)
     {
-      LM_T(LmtMongo, ("Orion database found: %s", dbName.c_str()));
-      dbsP->push_back(dbName);
-      LM_T(LmtBug, ("Pushed back db name '%s'", dbName.c_str()));
+      // Check for size of database name, db names size should be <= '50' bytes.
+      if (strlen(dbName.c_str()) <= 50)
+      {
+        LM_T(LmtMongo, ("Orion database found: %s", dbName.c_str()));
+        dbsP->push_back(dbName);
+        LM_T(LmtBug, ("Pushed back db name '%s'", dbName.c_str()));
+      }
+      else
+      {
+        LM_E(("Runtime Error (database name size should be smaller than 50 characters: %s)", dbName.c_str()));
+        return false;
```

Comment:
```suggestion
```

We don't want a "bad" database ruins all the cache refresh. Just skipping it (with the proper LM_E error) is fine, and the for loop will continue.

Reply:
Fixed in [8926692](https://github.com/telefonicaid/fiware-orion/pull/4297/commits/8926692de1eb33c90acbefc49dcd180488cba335)